diff -Nur gobject-introspection-0.6.0.orig/configure.ac gobject-introspection-0.6.0.libtool/configure.ac
--- gobject-introspection-0.6.0.orig/configure.ac	2008-10-31 14:01:42.000000000 -0600
+++ gobject-introspection-0.6.0.libtool/configure.ac	2008-11-22 13:38:23.000000000 -0700
@@ -76,7 +76,7 @@
 
 AC_MSG_CHECKING(for the suffix of shared libraries)
 export SED
-shrext_cmds=`./libtool --config | grep '^shrext_cmds='`
+shrext_cmds=`libtool --config | grep '^shrext_cmds='`
 eval $shrext_cmds
 eval std_shrext=$shrext_cmds
 # chop the initial dot
